internal/drivers/tart/driver.go

```diff
@@ -0,0 +1,733 @@
+package tart
```

Comment:
@raghavharness this seems like a big duplication of the nomad driver. How much is the delta and can we not consolidate all the existing logic in one place? I imagine only the startup scripts etc might be different?

We could separate it into two interfaces - one a scheduling layer and one a virtualization layer. Virtualization layer could be ignite/tart and scheduling layer would be common - nomad.

Reply:
@vistaarjuneja added suggested changes, can you check once.